schemas/string_interpolation/definitions.json

```diff
@@ -0,0 +1,143 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "string_with_placeholders": {
+    "oneOf": [
```

Comment:
We should have additional properties false at this level to avoid odd behaviour when using string type.
```json
"string_with_placeholders": {
	"additionalProperties": false,
	"oneOf": [
		...
```